### PR TITLE
chore: update vue2 example using experiments.css

### PR DESCRIPTION
**Pull request by hardfist**
## Summary
fix https://github.com/web-infra-dev/rspack/issues/3434

## Test Plan
No Need

## Require Documentation?

- [ ] No
- [x] Yes, the corresponding rspack-website PR is (waiting for us backport it to official vue-loader)

### CODE REVIEW

File: examples/vue2/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "less": "4.1.3",
     "less-loader": "^11.1.0",
     "vue": "2.7.14",
-    "vue-loader": "^15.10.1",
+    "vue-loader": "npm:@rspack/loader-vue2@15.10.1-alpha.0",
```
Comment (chenjiahan) [inline examples/vue2/package.json:17]: Can @rspack/loader-vue2 release a stable version?

**Review by chenjiahan — APPROVED** [review]

File: examples/vue2/package.json (hunk 1 above)
Comment (hardfist) [reply examples/vue2/package.json:17]: I'm not sure, cc @h-a-n-a , and the author file PR to vue-loader repo, when it's merged we can replace it with official vue-loader

Comment (h-a-n-a) [reply examples/vue2/package.json:17]: I don't know if it is a good choice to file a PR to vue, but seems like the target branch should be set to v15

Comment (hardfist) [reply examples/vue2/package.json:17]: I  talked with soda about this, he's willing to accept the PR

Comment (sodatea) [reply examples/vue2/package.json:17]: Reviewed and published as https://github.com/vuejs/vue-loader/releases/tag/v15.11.0

Comment (chenjiahan) [reply examples/vue2/package.json:17]: Thanks!

**Pull request merged**